xclim/indices/_multivariate.py

```diff
@@ -61,26 +61,26 @@
 ]
 
 
-@declare_units(tasmin="[temperature]", tn10="[temperature]")
+@declare_units(tasmin="[temperature]", per_da="[temperature]")
 @percentile_bootstrap
 def cold_spell_duration_index(
     tasmin: xarray.DataArray,
-    tn10: xarray.DataArray,
+    per_da: xarray.DataArray,
```

Comment:
Oh! I wrote the comment above before seeing that you were renaming the inputs. I think the idea from above still holds, but I indeed agree with generalizing the indices by removing the percentile from the name.

I believe you missing a few cases in the examples, but the idea is good.
I think it should be done everywhere and that includes things like `cold_and_dry_days`, and thus maybe the thresholded variable's name should be included in the name? `tas_per` ? Only for the multivariate cases?

Reply:
I would also vote for something similar `tasmin_percentile` or `tn_percentile` for this call signature. `percentile` alone give me the impression that I can pass `tas` (and not `tasmin`).

Reply:
Yep, and I forgot to rename in unit tests as well.
I'm ok with `tasmin_per` here and `tas_per`, `tasmax_per`, `pr_per` where needed. I think it's better to apply the same logic everywhere, it makes the code much easier to maintain.

BTW, I forgot to add it in the initial PR statement but renaming these is a breaking changes, is okay ?

Reply:
Changing call signatures is OK by me, but be sure to mention them in the History, and bump the version as well!